CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ## [Unreleased]
 
+## [v1.3.0]
+1.3.0 is a backwards compatible feature release. This module release supports Forseti v2.13.0.
```

Comment:
The change to the default value of `var.forseti_version` should be logged under CHANGED.

Note for posterity: the format of this file should be changed to match Keep a Changelog.

Reply:
Fixed in 110908e.

Following up on the format changes, what diff would you like to see?

Reply:
I think it's just the uppercase headers, which is admittedly ultra aesthetic. :blush: